Client/Frontend/Browser/OpenSearch.swift

```diff
@@ -47,10 +61,11 @@ class OpenSearchEngine {
 
     private func getURLFromTemplate(searchTemplate: String, query: String) -> NSURL? {
         let allowedCharacters = NSCharacterSet(charactersInString: SearchTermsAllowedCharacters)
-
         if let escapedQuery = query.stringByAddingPercentEncodingWithAllowedCharacters(allowedCharacters) {
             let urlString = searchTemplate.stringByReplacingOccurrencesOfString("{searchTerms}", withString: escapedQuery, options: NSStringCompareOptions.LiteralSearch, range: nil)
-            return NSURL(string: urlString)
+            if let encodedUrlString = urlString.stringByAddingPercentEncodingWithAllowedCharacters(NSCharacterSet.symbollessURLAllowedSet()) {
```

Comment:
> The issue is with the 特 character. Try passing that url string into NSURL and see if it returns an NSURL object. I think it returns nil because NSURL blows up when trying to parse it.

Hm, that code snippet above works for me (non-nil URL). Do you see something different?